vsintegration/src/FSharp.Editor/Navigation/GoToDefinition.fs

```diff
@@ -818,6 +811,7 @@ type internal FSharpNavigation
 
     member _.TryGoToDefinition(position, cancellationToken) =
         let gtd = GoToDefinition(metadataAsSource)
+        let statusBar = StatusBar()
```

Comment:
So, here (and elsewhere) - are you sure creating a status bar from scratch is equivalent to passing around the thing that flows in?

Reply:
Looking again at this, IMV we should just yeet all the statusbar code. It does nothing anyway.

Reply:
Looks like it was just a clunky wrapper around SVsStatusbar service, when it worked, pretty much stateless on its own, so creating it ad hoc is ok. Removing this completely looks like serious work, this file is 1000+ freaking lines🙃